components/teams-service/storage/postgres/postgres.go

```diff
@@ -96,6 +93,25 @@ func (p *postgres) StoreTeamWithProjects(ctx context.Context,
 	return team, nil
 }
 
+func (p *postgres) validateTeamInputs(ctx context.Context, name string, description string, projects []string) error {
+	if emptyOrWhitespaceOnlyRE.MatchString(name) {
```

Comment:
why do we need this regex check if it's already being validated in the proto?

Reply:
The whitespace only is not being validated in the proto, (because of the ugly message I mentioned).